client/components/place/place-list/place-list.template.html

```diff
@@ -298,81 +296,200 @@
+                    Name is required!
+                  </span>
+                </div>
+              </div>
+              <div class="form-group col-lg-6 col-md-12">
+                <label for="newPointType">
+                  <span class="title">Point type:</span>
+                </label>
+                <select name="newPointType" class="form-control"
+                        id="newPointType"
+                        ng-options="type as typeObj.name for (type, typeObj) in $ctrl.placesType track by type"
+                        ng-model="$ctrl.newPoint.type" ng-required="true">
+                  <option value="">-- Point type --</option>
+                </select>
+                <div ng-show="pointMaker.$submitted || pointMaker.newPointType.$touched">
+                  <span class="alert-input"
+                        ng-show="pointMaker.newPointType.$error.required">
+                    Type is required!
+                  </span>
+                </div>
+              </div>
+              <div class="form-group col-xs-12">
+                <label for="newPointDescription">
+                  <span class="title">Point description (optional):</span>
+                </label>
+                <textarea name="newPointDescription" class="form-control"
+                          id="newPointDescription"
+                          ng-model="$ctrl.newPoint.description">
+                </textarea>
+              </div>
+              <div class="form-group visible-lg">
+                <div class="btn-group col-xs-12">
+                  <input type="button" class="btn btn-success col-lg-6"
+                         ng-click="$ctrl.cancelNewPointForTrackMenu(pointMaker, true)"
+                         value="Cancel">
+                  <input type="submit" class="btn btn-success col-lg-6"
+                         value="Create Point"> 
+                </div>
+              </div>
+              <div class="form-group hidden-lg">
+                <div class="btn-group-vertical col-xs-12">
+                  <input type="button" class="btn btn-success col-md-12"
+                         ng-click="$ctrl.cancelNewPointForTrackMenu(pointMaker, true)"
+                         value="Cancel">
+                  <input type="submit" class="btn btn-success col-md-12"
+                         value="Create Point"> 
+                </div>
+              </div>
+            </form>
           </div>
-          <div class="form-group col-xs-12">
-            <label for="trackName"><span
-                    class="title">2. Enter track name:</span></label>
+        </div>   
+        <h2>New Track</h2>
+        <a href="" ng-click="$ctrl.toggleAddTrackMenu(trackMaker)">
+          <span class="glyphicon glyphicon-remove removeIcon"></span>
+        </a>
+        <span class="title">1. Choose places on map:</span><br>
+        <div id="pointsForNewTrack">
+          <table ng-repeat="point in $ctrl.newTrackPoints">
```

Comment:
It's better to use bootstraps `row` here.
Table tag should only be used for actual tables (which contains data)